{{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/cronjob.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: cron-backup
+spec:
+  schedule: "1 1 * * *"
```

Comment:
Feels like a magic number. How about something more self-explanatory? (e.g. `"59 1 * * *"` would at least convey that the first value is "minutes")

Reply:
OK, I updated the value.